py/desisurvey/nightops.py

```diff
@@ -14,6 +14,10 @@
 from surveysim.observefield import observeField
 import desisurvey.nightcal
 
+LSTres = 1.0/144.0 # Should be the same as in afternoon planner and next field selector
```

Comment:
This is a bit confusing because units are days here but deg in afternoonplan, but otherwise this seems ok.